.gitbook/README.md

```diff
@@ -16,22 +16,11 @@ layout:
 
 This is the primary place where [Boba](https://boba.network) works on the Boba L2, a compute-focused L2. Fundamentally, Ethereum is a distributed computer. We believe that L2s can play a unique role in augmenting the base _compute_ capabilities of the Ethereum ecosystem. You can learn more about Turing hybrid compute [here](contents/hc/README.md).
 
-<figure><img src="./assets/Hybrid Compute page - Technical details chart.png" alt=""><figcaption></figcaption></figure>
-
-Boba is built on the Optimistic Rollup developed by [Optimism](https://optimism.io). Aside from it focus on augmenting compute, Boba differs from Optimism by:
-
-* providing additional cross-chain messaging such as a `message-relayer-fast`
-* using different gas pricing logic
-* providing a swap-based system for rapid L2->L1 exits (without the 7 day delay)
-* providing a community fraud-detector that allows transactions to be independently verified by anyone
-* interacting with L2 ETH using the normal ETH methods (`msg.value`, `send eth_sendTransaction`, and `provider.getBalance(address)` rather than as WETH
-* being organized as a [DAO](https://github.com/bobanetwork/boba\_legacy/tree/docs-in-monrepo/packages/boba/contracts/contracts/DAO)
-* native [NFT bridging](https://github.com/bobanetwork/boba\_legacy/tree/docs-in-monrepo/packages/boba/contracts/contracts/bridges)
-* automatically relaying classical 7-day exit messages to L1 for you, rather than this being a separate step
+Boba is built on the Optimistic Rollup developed by [Optimism](https://optimism.io). In addition to the native features of Optimism like bridging ETH and other tokens (including NFTs), Boba extends Optimism with compatible extensions including [Hybrid Compute](contents/hc/README.md) and [Boba as a Fee Token](contents/developer/features/aa-basics/aa-paymasters.md).  Boba is organized as a [DAO](https://gateway.boba.network/dao).
 
 <figure><img src="./assets/documentation.png" alt=""><figcaption></figcaption></figure>
 
-User focused documentation is available [on the Boba docs website](http://docs.boba.network/). If you have questions or feel like something is missing check out our [Discord server](https://discord.com/invite/YFweUKCb8a) where we are actively responding, or [open an issue](https://github.com/bobanetwork/boba/issues) in the GitHub repo for this site.
+User focused documentation is available [here on the Boba docs website](http://docs.boba.network/). If you have questions or feel like something is missing check out our [Discord server](https://discord.com/invite/YFweUKCb8a) where we are actively responding, or [open an issue](https://github.com/bobanetwork/boba/issues) in the GitHub repo for this site.
```

Comment:
the discord server invite has expired

Reply:
Good catch, fixed here, and in two other places as well